app/views/service_requests/right_navigation/_right_navigation.html.haml

```diff
@@ -20,7 +20,10 @@
 = javascript_include_tag 'right_navigation'
 
 = faq_helper
-= feedback_helper
+- if USE_REDCAP_API
+  = link_to 'Feedback', new_feedback_path, remote: true, class: 'btn btn-primary btn-lg btn-block feedback-button'
+- else
+  = feedback_helper
```

Comment:
In feedback_helper, there's a call to USE_FEEDBACK_LINK and FEEDBACK_URL which was a link to the Redcap Survey. It should probably be removed, and you could remove the helper and put a simple tag here.